src/introspection/memory/ffi.rs

```diff
@@ -123,11 +123,12 @@ pub struct MemPoolIntrospectionTopic {
 }
 
 impl MemPoolIntrospectionTopic {
-    pub fn new() -> Topic<Self> {
-        TopicBuilder::<Self>::new("Introspection", "RouDi_ID", "MemPool")
+    pub fn new() -> InactiveSubscriber<Self> {
+        SubscriberBuilder::<Self>::new("Introspection", "RouDi_ID", "MemPool")
             .queue_capacity(1)
             .history_request(1)
-            .build()
+            .create_without_subscribe()
```

Comment:
Just out of curiosity, we have `create`, `create_mt` (for multithreaded), `create_without_subscriber`, `create_without_subscribe_mt`?

The point I wanna make is, I would put everything into the builder and just have a single create so it is nice and consistent and I remember a special someone nagging me about consistency with `m_` so I want to return the favor :smile: 

So the handling I imagine is something like this:
```rust
    pub fn new() -> InactiveSubscriber<Self> {
        SubscriberBuilder::<Self>::new("Introspection", "RouDi_ID", "MemPool")
            .queue_capacity(1)
            .history_request(1)
            .threadsafe(false)
            .subscribe_on_create(false)
            .build()
            .create_without_subscribe()
```

Reply:
But then you don't need a builder or you need a second builder for `InactiveSubscriber`since `new` does not have any parameter. I explained the rationale further down.